uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm

```diff
@@ -1352,18 +1729,16 @@ sub orderModalitiesByAcq {
     }
 }
 
-################################################################
-################ getUploadIDUsingTarchiveSrcLoc#################
-################################################################
 =pod
-getUploadIDUsingTarchiveSrcLoc()
-Description:
-  - Get upload_id form the mri_upload table using tarchive SourceLocation
 
-Arguments:
-  $tarchive_srcloc: The Tarchive SourceLocation
+=head3 getUploadIDUsingTarchiveSrcLoc($tarchive_srcloc)
+
+Gets the upload ID form the mri_upload table using tarchive's SourceLocation
```

Comment:
please use C<> for table names `mri_upload` and `tarchive`; and the column `SourceLocation`